### PR TITLE
NGVALID-663 : Add Debian docker image and builds to ngraph-tf

### PR DESCRIPTION
**Pull request by lnguyen-nvn**
- Pull debian:buster  from hub docker, and its env are:
# with gcc (Debian 8.2.0-21) 8.2.0 (or latest)
# with **clang-format-6.0**
# with python 3.5

High-light **the issues** need to be fixed before  merge this PR:

1. Fix  clang-format version.
a. clang-format-3.9 is not supported in Debian , so I use clang-format-6.0. Need to update  clang-format in check-code-format.sh
b. 8 of 57 source files failed the code-format check:
[Build-NGTF-Testing]     src/ngraph_conversions.h
[Build-NGTF-Testing]     src/ngraph_encapsulate_op.cc
[Build-NGTF-Testing]     src/ngraph_builder.cc
[Build-NGTF-Testing]     src/ngraph_assign_clusters.cc
[Build-NGTF-Testing]     src/ngraph_encapsulate_clusters.cc
[Build-NGTF-Testing]     test/test_math_ops.cpp
[Build-NGTF-Testing]     test/opexecuter.cpp
[Build-NGTF-Testing]     test/test_array_ops.cpp

Moreover, python3.7 doesn't work with upstream https://github.com/RasaHQ/rasa_nlu/issues/1709 . Therefore, I use python3.5. 

Please review NGVALID-663 for more detail information (as testing result). 

As @crlishka mentioned, this is P2 ticket.

### CODE REVIEW

File: test/ci/docker/dockerfiles/Dockerfile.ngraph_tf.build_ngtf_debian_gcc6_py35 [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+# ==============================================================================
+
+# Environment to build and unit-test ngraph-tensorflow-bridge
+
+# Environment to build and unit-test ngraph on debian10 (buster)
```
Comment (crlishka) [inline test/ci/docker/dockerfiles/Dockerfile.ngraph_tf.build_ngtf_debian_gcc6_py35]: Could you please update the comments to reflect the change to debian:stretch?  Thanks.

Comment (lnguyen-nvn) [reply test/ci/docker/dockerfiles/Dockerfile.ngraph_tf.build_ngtf_debian_gcc6_py35]: Oh. Sorry !!

**Comment by crlishka** [comment]
I have removed the WIP tag, as the Dockerfile changes look good to me.  This PR is ready for review by @avijit-nervana

**Comment by crlishka** [comment]
Inspection of the commits shows that a lot were lost during the "accident" last Friday.  That acknowledge, I have a private clone of the repo branch with the later changes, so in the end nothing was fully lost.

After discussing with both @lnguyen-nvn and @avijit-nervana , I am going to close this PR because the ngraph-tf repo is being retired.  I will reopen this PR in the new ngraph-bridge repo, and have both Lam and Avijit as reviewers there.

**Pull request closed without merging**